Pinta.Core/Classes/AsyncEffectRenderer.cs

```diff
@@ -43,7 +42,9 @@ namespace Pinta.Core;
 // Only call methods on this class from a single thread (The UI thread).
 internal sealed class AsyncEffectRenderer
 {
-	private readonly Settings settings;
+	internal readonly record struct CompletionInfo (
+		bool WasCanceled,
+		ConcurrentQueue<Exception> Errors);
```

Comment:
I'd prefer changing this back to an `IReadOnlyList` or similar - returning a `ConcurrentQueue` is exposing some implementation details that the caller shouldn't care about

Reply:
@cameronwhite I changed it back to `IReadOnlyList<T>`.

What I did was intended to be an intermediate refactoring, but I understand if you are aiming to have decent code in `master` at all times.

Reply:
Thanks!